### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by pixELSflash**

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<resources
+        xmlns:tools="http://schemas.android.com/tools"
+        tools:ignore="MissingTranslation">
+        <string name="app_name">Kaffee</string>
+        <string name="keep_display_on">Bildschirm eingeschaltet lassen</string>
+        <string name="tap_to_turn_off">Tippen zum Ausschalten</string>
+        <string name="hint_1">Diese App stellt in den Schnelleinstellungen (Das Menü, wo man W-LAN und Bluetooth ein- und ausschalten kann) eine Kachel zur Verfügung, welche es erlaubt, den Bildschirm eingeschaltet zu lassen, ohne die Geräteeinstellungen zu ändern</string>
```
Comment (mueller-ma) [inline app/src/main/res/values-de/strings.xml:7]:
```suggestion
        <string name="hint_1">Diese App stellt in den Schnelleinstellungen (der Ort, an dem die Schalter für z.B. WLAN und Bluetooth sind) eine Kachel zur Verfügung, welche es erlaubt, den Bildschirm eingeschaltet zu lassen, ohne die Geräteeinstellungen zu ändern</string>
```

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        <string name="hint_2">Kaffee kann durch Klicken auf den Titel oder die Benachrichtigung oder das manuelle Ausschalten des Bildschirms gestoppt werden</string>
```
Comment (mueller-ma) [inline app/src/main/res/values-de/strings.xml:8]:
```suggestion
        <string name="hint_2">Kaffee kann durch Tippen auf die Kachel oder die Benachrichtigung oder durch das manuelle Ausschalten des Bildschirms gestoppt werden</string>
```

**Review by mueller-ma — COMMENTED** [review]
Thanks for your translation. Can you check my suggestions?

File: app/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (pixELSflash) [reply app/src/main/res/values-de/strings.xml:7]: That definitely sounds nicer. I was unsure about that myself but couldn't think of anything better

File: app/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (pixELSflash) [reply app/src/main/res/values-de/strings.xml:8]: I must have misread the english text. Thanks for correcting

**Pull request merged**